airbyte-integrations/connectors/source-mailjet-mail/source_mailjet_mail/spec.yaml

```
@@ -0,0 +1,21 @@
+documentationUrl: https://dev.mailjet.com/email/guides/
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Mailjet Mail Spec
+  type: object
+  required:
+    - api_key
+    - api_key_secret
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: >-
+        Your API Key. See <a
+        href="https://dev.mailjet.com/email/guides/#authentication">here</a>.
```

```suggestion
    api_key:
      title: API Key
      type: string
      description: >-
        Your API Key. See <a
        href="https://dev.mailjet.com/email/guides/#authentication">here</a>.
```

Please add title to your parameters.